playbooks/example-source/group_vars/all

```diff
@@ -10,6 +10,8 @@ pulp_install_plugins:
   #   source_dir: "/var/lib/pulp/devel/pulp_ansible"
   # pulp-certguard:
   #   source_dir: "/var/lib/pulp/devel/pulp-certguard"
+  pulp-cli:
+    source_dir: "/var/lib/pulp/devel/pulp-cli"
```

Comment:
Wondering, this actually works. Technically it is not a plugin. Maybe we don't do anything else but pip install plugins...
Still don't we have a section for extra_pip_packages?

Reply:
we have `prereq_pip_packages`, I think we can do `extra_pip_packages`,
I put it here `pulp_install_plugins` because I was unsure if pulp-cli can be considered as a plugin or not